xml/System.Threading/Volatile.xml

```diff
@@ -982,7 +985,7 @@
       </Parameters>
       <Docs>
         <param name="location">The field where the value is written.</param>
-        <param name="value">The value to write. The value is written immediately so that it is visible to all processors in the computer.</param>
```

Comment:
is the extra clarification not required, or has the behavior now changed?

Reply:
The behavior hasn't changed, just removing the statement since it's incorrect